gdctools/gdc_mirror.py

```diff
@@ -45,6 +46,8 @@ def __init__(self):
         cli.add_argument('-f', '--force-download', action='store_true',
                 help='Download files even if already mirrored locally.'+
                 ' (DO NOT use during incremental mirroring)')
+        cli.add_argument('--append', default=False, action='store_true',
+                help='')
```

Comment:
Please add a help description for --append

Reply:
also, when `action` is set to `'store_true'` the default is automatically set to `False`

Reply:
Yes, docs in general.